src/routes/api/index.js

```diff
@@ -1,18 +1,7 @@
-const router = require("express").Router();
+import express from 'express';
+import users from './authRoute';
 
-router.use("/", require("./users"));
+const router = express.Router();
+router.use('/auth', users);
```

Comment:
Then change this users here to authRoute for name convention

Reply:
Corrected